core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -140,6 +140,7 @@ public CallOptions withCompressor(@Nullable Compressor compressor) {
    * TLS certificate.</em>
    */
   @Nullable
+  @ExperimentalApi
```

Comment:
This is being marked experimental because `withAuthority()` is experimental, no reason more. I added a link to both of them.